### PR TITLE
feat: Implement logic to create orgs

### PR DESCRIPTION
**Pull request by levansuper**
# What kind of change does this PR introduce?
 

# What is the current behavior?
 

# What is the new behavior (if this is a feature change)?

# Other information

### CODE REVIEW

File: src/services/routes/org.routes.spec.ts
Comment (gnarea) [inline src/services/routes/org.routes.spec.ts:31]:
```suggestion
    test('Valid parameters should return URLs', async () => {
```

Just for consistency with other repos.

File: src/services/routes/org.routes.ts
Comment (gnarea) [inline src/services/routes/org.routes.ts:11]: I'm sure you tried this, but just to double check: isn't there a simpler alternative to `(typeof HTTP_STATUS_CODES)[keyof typeof HTTP_STATUS_CODES]`?

Doesn't `keyof typeof HTTP_STATUS_CODES` work?

File: src/services/routes/org.routes.ts
Comment (gnarea) [inline src/services/routes/org.routes.ts:10]: Since this isn't exported, I'd suggest something along the lines of `RESPONSE_CODE_BY_PROBLEM_TYPE` or `RESPONSE_CODE_BY_PROBLEM`.

File: src/services/routes/org.routes.ts
Comment (gnarea) [inline src/services/routes/org.routes.ts:22]: I'd say "form" is correct but it could potentially be ambiguous. How about `makeUrls()`?

File: src/services/routes/org.routes.ts
Comment (gnarea) [inline src/services/routes/org.routes.ts:47]: Can you try dropping this? It shouldn't be needed -- plus, Fastify would automatically specify the charset, which is a good idea.

File: src/services/routes/org.routes.ts
Comment (gnarea) [inline src/services/routes/org.routes.ts:54]: Same here.

**Review by gnarea — CHANGES_REQUESTED** [review]

**Comment by gnarea** [comment]
FYI @levansuper, I just changed the title to make it more user-friendly, since this will go on the release notes.

File: src/services/routes/org.routes.ts
Comment (levansuper) [reply src/services/routes/org.routes.ts:11]: Yep I tried that.
 keyof typeof HTTP_STATUS_CODES returns the keys of the object like "OK", "NO_CONTENT", etc...
 In this case we need the values not the keys that's why I had to write this

Comment (gnarea) [reply src/services/routes/org.routes.ts:11]: Cool, let's leave it as it is then 👍🏾

File: src/services/routes/org.routes.ts
Comment (levansuper) [reply src/services/routes/org.routes.ts:54]: OK is very specific for the success case. we will not have reason in this case

Comment (gnarea) [reply src/services/routes/org.routes.ts:54]: Note I'm referring to the `Content-Type`... I think that if we drop that line, Fastify will use `application/json; charset=utf-8`, which is even better.

**Review by gnarea — DISMISSED** [review]
LGTM but CI is still blocked until you fix the merge conflicts with `main`.

**Review by gnarea — APPROVED** [review]
Awesome, thanks! 🚀

**Pull request merged**
